### PR TITLE
Use Timber mpp

### LINKED ISSUE #274: use Timber Mpp?
## Overview (Required)

timber supports multi platform from 5.0.0-SNAPSHOT. How about using it?

pros
- Timber Android ecosystem is so large.
- Timber has a flexible API.

cons
- snapshot... not stable.
- iOS sample implements not exists in Timber project. So we must be implemented code for iOS Timber log.

## Links
- Timber: https://github.com/JakeWharton/timber
- SdkSearch: https://github.com/JakeWharton/SdkSearch

### PR DESCRIPTION
**Pull request by satoshun**
## Issue
- close #274 

## Overview (Required)
- Timber support MPP from 5.0.0 snapshot, i try it!

## Links
- https://github.com/JakeWharton/timber

## Screenshot
Before | After
:--: | :--:
 |

### CODE REVIEW

File: ext/log/src/commonMain/kotlin/io/github/droidkaigi/confsched2019/timber/Timber.kt [hunk 1]
```diff
@@ -0,0 +1,29 @@
+@file:Suppress("NOTHING_TO_INLINE")
+
+package io.github.droidkaigi.confsched2019.timber
+
+import timber.log.Timber
+
+inline fun Timber.assert(throwable: Throwable) {
```
Comment (satoshun) [inline ext/log/src/commonMain/kotlin/io/github/droidkaigi/confsched2019/timber/Timber.kt]: Timber doesn't have helper method without message String. i think, it's helper method useful.

File: feature/session/build.gradle [hunk 2]
```diff
@@ -49,6 +48,8 @@ dependencies {
     implementation Dep.Picasso.picasso
     implementation Dep.Picasso.picassoTransformation
 
+    implementation Dep.Timber.android
```
Comment (satoshun) [inline feature/session/build.gradle]: compile error is occur if it's removed. but i add dependency of Timber in ":ext:log" module. kotlin-native doesn't support Gradle dependency api?

File: ext/log/build.gradle [hunk 3]
```diff
@@ -12,10 +12,12 @@ kotlin {
     sourceSets {
         commonMain.dependencies {
             api Dep.Kotlin.stdlibCommon
+            api Dep.Timber.common
         }
         androidMain {
             dependencies {
                 api Dep.Kotlin.stdlibJvm
+                api Dep.Timber.jdk
```
Comment (satoshun) [inline ext/log/build.gradle]: i add Timber dependency in it. but client module needs explicitly Timber dependency.

File: ext/log/src/commonMain/kotlin/io/github/droidkaigi/confsched2019/timber/Timber.kt (hunk 1 above)
Comment (takahirom) [reply ext/log/src/commonMain/kotlin/io/github/droidkaigi/confsched2019/timber/Timber.kt]: It's useful!!

**Comment by takahirom** [comment]
LGTM 🎄

**Pull request merged**